src/config.ts

```diff
@@ -23,26 +23,42 @@ const handleSummary = (summary: K6Summary) => {
 
   return {
     [`.results/${Date.now()}.json`]: JSON.stringify(_summary), //the default data object
-    stdout: textSummary(summary, { enableColors: false })
+    stdout: textSummary(summary, { enableColors: false }),
   };
 };
 
-export const defaultOptions: Options = {
-  stages: [
-    { duration: "10s", target: 5 }, // simulate ramp-up of traffic from 1 to 5 users.
-    { duration: "10s", target: 5 }, // stay at 5
-    { duration: "10s", target: 10 }, // ramp-up to 10 users
-    { duration: "10s", target: 10 }, // stay at 10 users for short amount of time
-    { duration: "10s", target: 5 }, // ramp-down to 5 users
-    { duration: "10s", target: 0 }, // ramp-down to 0 users
-  ],
-};
+// export const defaultOptions: Options = {
+//   stages: [
+//     { duration: "10s", target: 5 }, // simulate ramp-up of traffic from 1 to 5 users.
+//     { duration: "10s", target: 5 }, // stay at 5
+//     { duration: "10s", target: 10 }, // ramp-up to 10 users
+//     { duration: "10s", target: 10 }, // stay at 10 users for short amount of time
+//     { duration: "10s", target: 5 }, // ramp-down to 5 users
+//     { duration: "10s", target: 0 }, // ramp-down to 0 users
+//   ],
+// };
 
+// For local testing purposes
 // export const defaultOptions: Options = {
 //   vus: 1,
 //   duration: "2s",
 // };
 
+// Constant rate of requests with a fixed number of VUs.
+export const defaultOptions: Options = {
+  scenarios: {
+    constant_request_rate: {
+      executor: "constant-arrival-rate",
+      // Number of iterations to execute each timeUnit period.
+      rate: 50,
+      timeUnit: "1s",
+      duration: "1m",
+      preAllocatedVUs: 25,
+      maxVUs: 25,
```

Comment:
this could be too small
if response time is slow you will run out of VUs trying to get 50 req/sec
id bump that to like 500